app/src/app/views/stix/collection/collection-import/collection-import-error/collection-import-error.component.html

```diff
@@ -1,26 +1,54 @@
 <div class="collection-import-error">
-    <p class="nomargin">
-        <span class="warn">WARNING</span>: the data you are trying to import contains some problems which may affect your knowledge base if you proceed with the import:
-    </p>
+    <ng-container *ngIf="hasWarnings">
+        <p class="nomargin">
+            <span class="warn">WARNING</span>: the data you are trying to import contains some problems which may affect your knowledge base if you proceed with the import:
+        </p>
 
-    <div class="error-list">
-        <ul>
-            <li *ngIf="duplicateCollection">
-                <span>The bundle has previously been imported. Some objects will not be imported.</span>
-            </li>
-            <li *ngIf="objectSpecVersionViolation > 0">
-                <span>{{objectSpecVersionViolation}} {{objectSpecVersionViolation > 1 ? 'objects use' : 'object uses'}} a newer version of the ATT&CK spec than your Workbench supports and will not be imported. After upgrading Workbench to the latest version you can re-import the data to acquire {{objectSpecVersionViolation > 1 ? 'these objects' : 'this object'}}. </span>
-            </li>
-        </ul>
-    </div>
+        <div class="warning-list">
+            <ul>
+                <li *ngIf="duplicateCollection">
+                    <span>The collection bundle already exists in the database. Some objects may not be imported.</span>
+                </li>
+                <li *ngIf="duplicateObjects > 0">
+                    <span>{{duplicateObjects}} {{duplicateObjects > 1 ? 'objects are' : 'object is'}} duplicated in the collection bundle and will not be imported.</span>
+                </li>
+                <li *ngIf="objSpecVersionViolations > 0">
+                    <span>{{objSpecVersionViolations}} {{objSpecVersionViolations > 1 ? 'objects use' : 'object uses'}} a newer version of the ATT&CK spec than your Workbench supports and will not be imported. After upgrading Workbench to the latest version you can re-import the data to acquire {{objSpecVersionViolations > 1 ? 'these objects' : 'this object'}}. </span>
+                </li>
+                <li *ngIf="objSpecViolations > 0">
+                    <span>{{objSpecViolations}} {{objSpecViolations > 1 ? 'objects have' : 'object has'}} other spec violations and will not be imported.</span>
+                </li>
+            </ul>
+        </div>
+    </ng-container>
+
+    <ng-container *ngIf="hasErrors">
+        <p class="nomargin">
+            <span class="error">ERROR</span>: the data you are trying to import contains errors and may not be imported:
```

Comment:
Exceedingly nitpicky, but:

> the data you are trying to import contains errors and may not be imported:

Should be changed to:

> the data you are trying to import contains errors and cannot be imported:

This is because `can`/`cannot` has to do with ability or capacity to do something whereas `may`/`may not` is more regarding permission to do something. They are oftentimes interchangeable, and certainly could be used interchangeably in this case, but I feel like `can` is more reflective to the lack of ability to bypass the error.